lib/runners/workspace.rb

```diff
@@ -89,7 +86,7 @@ def self.prepare_in_dir(type, source, key, dir, trace_writer:)
         Retryable.retryable(
           tries: 5,
           on: [OpenSSL::Cipher::CipherError, OpenSSL::SSL::SSLError, DownloadError, Net::OpenTimeout, Errno::ECONNRESET, SocketError, Net::HTTPForbidden],
-          sleep: -> (n) { n + 1 },
+          sleep: retryable_sleep,
```

Comment:
[question]

1. Do we need to wrap with [`method`](http://ruby-doc.org/core-2.6.5/Object.html#method-i-method) as follow?

```suggestion
          sleep: method(:retryable_sleep),
```

2. I feel that `attr_reader :retryable_sleep` is a bit verbose and confusing. How about adding a private method like this?

```ruby
def retryable_sleep(n)
  n + 1
end
```

Reply:
If I continue using the `:retryable_sleep` as a `Proc` value, it seems to work without `method`, but declaring it as an instance method looks good. I'll fix it as you said.